go.sum

```diff
@@ -470,6 +470,8 @@ github.com/tmc/grpc-websocket-proxy v0.0.0-20190109142713-0ad062ec5ee5/go.mod h1
 github.com/tmc/grpc-websocket-proxy v0.0.0-20201229170055-e5319fda7802/go.mod h1:ncp9v5uamzpCO7NfCPTXjqaC+bZgJeR0sMTm6dMHP7U=
 github.com/wind-river/gophercloud v0.0.0-20220714135506-aac06588b172 h1:xSeAummWb7dffFtdHq+Tqk1VeMdvxzk5N5k1m8M9wDA=
```

Comment:
Please remove the redundant references here

Reply:
Removed the old reference